pwiz_tools/Skyline/SettingsUI/ViewLibraryDlg.cs

```diff
@@ -813,17 +828,40 @@ public void UpdateUI(bool selectionChanged = true)
                                 .GetSpectra(_peptides[index].Key, null, LibraryRedundancy.best).FirstOrDefault();
                         }
 
-                        var spectrumInfoR = LibraryRankedSpectrumInfo.NewLibraryRankedSpectrumInfo(spectrumInfo.SpectrumPeaksInfo,
-                                                                          transitionGroupDocNode.TransitionGroup.LabelType,
-                                                                          transitionGroupDocNode,
-                                                                          settings,
-                                                                          transitionGroupDocNode.Peptide.Target,
-                                                                          mods,
-                                                                          adducts,
-                                                                          types,
-                                                                          rankCharges,
-                                                                          rankTypes,
-                                                                          (spectrumInfo?.SpectrumHeaderInfo as BiblioSpecSpectrumHeaderInfo)?.Score);
+
+                        // Generates the object that will go into the property sheet
+                        var newProperties = new SpectrumProperties()
+                        {
+                            RetentionTime = string.Format(@"{0:F2}", spectrumInfo.RetentionTime),
+                            FileName = spectrumInfo.FileName,
+                            LibraryName = spectrumInfo.Name,
+                            PrecursorMz = string.Format(@"{0:F2}", CalcMz(pepInfo, _matcher)),

```

Comment:
2 decimal places seems short of m/z. Research what we are using elsewhere in the project.

Reply:
In the same function below, we use the following for m/z (4 optional decimal places):
chromData.Mz.ToString(@"0.####")